docs/UserGuide.md

```diff
@@ -116,105 +118,124 @@ Note that each command might use the prefixes slightly differently so refer to e
 **Notes:**
 
 1. Amounts can be exactly `$0.00` as users may want to simply track that a transaction is present but not specify the amount.
+User might also want to track financial events not involving currency exchange, such as barter trading, free gifts, etc. 
 2. Intervals work by filtering by the specified time period. 
    1. For `day` intervals, only transactions of the same day are found. 
    2. For `week` intervals, only transactions of the same [week of year](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/temporal/WeekFields.html#weekOfYear()) are found. 
    3. For `month` intervals, only transactions of the same month are found.
 
-### UI Layout
+[//]: # ()
 
-UniCa$h is designed with users who prefer to use the keyboard in mind. Thus, almost all
-user input is designed for CLI-type usage, i.e. text-based keyboard input, and UI elements are intended 
-to supplement this main functionality.
+[//]: # (UI layout and description of what each section means)
```

Comment:
Don't think these are necessary to keep in

Reply:
Yeah makes more sense for the whole of the UI Part to be in the DG instead.

Reply:
The paragraphs could also be moved to DG instead too, keep the UG brief